### PR TITLE
Turkish translation updates

### PR DESCRIPTION
**Pull request by kayazeren**
Turkish translation updates for WinMerge and ShellExtension

### CODE REVIEW

File: Translations/ShellExtension/Turkish.po [hunk 1]
```diff
@@ -1,15 +1,18 @@
-# This file is part from WinMerge <https://winmerge.org/>
+# This file is part from WinMerge <http://winmerge.org/>
```
Comment (sdottaka) [inline Translations/ShellExtension/Turkish.po:1]: Is this fix intended or an accidental fix?

Comment (kayazeren) [reply Translations/ShellExtension/Turkish.po:1]: Sorry accidental. I just missed. Should be https. Thanks.

Comment (sdottaka) [reply Translations/ShellExtension/Turkish.po:1]: Thank you for your reply.
I'll fix that later.

**Comment by sdottaka** [comment]
Thank you!

**Pull request merged**